cluster-autoscaler/clusterstate/clusterstate.go

```diff
@@ -1023,6 +1050,13 @@ func expectedToRegister(instance cloudprovider.Instance) bool {
 	return instance.Status != nil && instance.Status.State != cloudprovider.InstanceDeleting && instance.Status.ErrorInfo == nil
 }
 
+func provisionedTime(instance cloudprovider.Instance, currentTime time.Time) time.Time {
```

Comment:
This method is used in a single place and is totally non descriptive, please inline it instead.